public/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "short_name": "MOSIP Idp",
```

Comment:
it should MOSIP - OIDC Demo Health Portal

Reply:
short_name updated to MOSIP - OIDC Demo Health Portal